sympy/vector/deloperator.py

```diff
@@ -16,7 +16,7 @@ def __new__(cls, system=None):
                 feature="delop operator inside coordinate system",
                 useinstead="it as instance Del class",
                 deprecated_since_version="1.1",
-                issue=12858,
```

Comment:
> Since you already opened the other one with more information, should #12858 be closed? 

I did not information that you mentioned me in #12858  and I didn't notice this issue. It is the reason why I opened the other one. 
#12858 can be closed